src/pages/MuitoManga/main.ts

```diff
@@ -0,0 +1,79 @@
+import { pageInterface } from '../pageInterface';
+
+export const MuitoManga: pageInterface = {
+  name: 'MuitoMangá',
+  domain: 'https://muitomanga.com',
+  languages: ['Portuguese'],
+  type: 'manga',
+  isSyncPage(url) {
+    return utils.urlPart(url, 3) === 'ler' && utils.urlPart(url, 4) !== '';
+  },
+  isOverviewPage(url) {
+    return utils.urlPart(url, 3) === 'manga' && utils.urlPart(url, 4) !== '';
+  },
+  getImage() {
+    return j.$('.capaMangaInfo img').first().attr('data-src');
+  },
+  sync: {
+    getTitle(url) {
+      return j
+        .$('.widget-title')
+        .first()
+        .text()
+        .match(/Ler (.+) Capítulo/)!
+        .pop()!;
+    },
+    getIdentifier(url) {
+      return utils.urlPart(url, 4);
+    },
+    getOverviewUrl(url) {
+      return utils.absoluteLink(
+        j.$('a[href*="/manga"]').first().attr('href') || '',
+        MuitoManga.domain,
+      );
+    },
+    getEpisode(url) {
+      return Number(utils.urlPart(url, 5).replace('capitulo-', ''));
+    },
+    nextEpUrl(url) {
```

Comment:
If on the last chapter of a manga then it returns the url to the overview page. Here returning undefined would be the way to go.

Reply:
Done!